pkg/scripts/Vagrantfile

```diff
@@ -0,0 +1,52 @@
+Vagrant.require_version ">= 1.8.7"
+unless Vagrant.has_plugin?("vagrant-vbguest")
+  raise "Please install the vagrant-vbguest plugin by running `vagrant plugin install vagrant-vbguest`"
```

Comment:
maybe also introduce `vagrant-cachier` as well, so that it doesn't download the image every time and cache it for development purposes?

Reply:
added below